### PR TITLE
continuation of "handle http requests"

### PR DESCRIPTION
**Pull request by JulianaCP-v2**

### CODE REVIEW

**Review by dshen109 — COMMENTED** [review]
Could we add brief module docs to each of the files modified here? I am concerned at the lack of tests + documentation in this repo in general since it seems like Juliana and Eliav are the only people who know exactly what the scripts do.

**Comment by JulianaCP-v2** [comment]
> Could we add brief module docs to each of the files modified here? I am concerned at the lack of tests + documentation in this repo in general since it seems like Juliana and Eliav are the only people who know exactly what the scripts do.

yes, sure. Im actually planning to add a general readme for this repo, probably I will add it in some of the next PRs, We still have to reorganize the scripts so not sure when but definitely will be considered in this story

File: operator_toolkit/loadshed.py [hunk 1]
```diff
@@ -1,85 +1,74 @@
...
-    virtual_operator_url = basic_url(ServiceType.VIRTUAL_OPERATOR, site)
-
-    # basic rack control url
-    rack_control_url = basic_url(ServiceType.RACK_CONTROL, site)
-
     # step 1: start mode bsl mode on site
-    mode_bsl = f'{virtual_operator_url}/modes/bsl/start'
-    resp = requests.post(mode_bsl, auth=HTTPBasicAuth(AXIOM_API_AUTH_USER, AXIOM_API_AUTH_PASSWORD))
-    report["start_bsl_mode"] = f"Status Code: {resp.status_code}"
+    mode_bsl = Operator.generate_url(ServiceType.VIRTUAL_OPERATOR, site, attachment="/modes/bsl/start")
```
Comment (lucasmlofaro) [inline operator_toolkit/loadshed.py:22]: I would recommend the term `path` instead of `attachment`

**Review by lucasmlofaro — APPROVED** [review]
I'm ok with this for now, it's mostly cosmetic and organizational, so I think it's a good start. A couple of notes:
- I think I missed the introduction of this `Operator` class, I would like to think a bit more about this abstraction at some point. The essential question for me is, does such an object actually exist/does it make sense to model it this way?
- Generally utility classes like this (where we use only static methods) is actually not desirable OO design, but at least we're starting to organize things, so I'd say we shouldn't worry about it for now and we can we can refactor later.

**Review by lucasmlofaro — APPROVED** [review]

**Pull request merged**